src/main/java/atrium/AtriumClient.java

```diff
@@ -577,6 +577,28 @@ public Account readAccount(String userGUID, String accountGUID) {
 
   // TRANSACTION
 
+  /**
+   * Method used to categorize and describe a Transaction.
+   * @param transactions Required. An array of RawTransaction objects.
+   * @return CategorizedTransaction[]. This returns an array of CategorizedTransaction objects.
+   */
+  public CategorizedTransaction[] categorizeAndDescribeTransactions(JsonArray transactions) {
```

Comment:
Why a JsonArray and not something like `List<Transaction> transactions`?

Reply:
Updated. This approach is the way to go. 👍 

I originally went with `JsonArray` as it was the convention we were using in a few other endpoints in this library. I'll be getting up another PR soon to update the other endpoints.